web-app/js/portal/search/GeoSelectionPanel.js

```diff
@@ -33,30 +33,37 @@ Portal.search.GeoSelectionPanel = Ext.extend(Ext.Panel, {
         this.facetMap = new Portal.search.FacetMapPanel({
             initialBbox: Portal.app.appConfig.portal.initialBbox,
             mainMap: cfg.mapPanel,
-            height: 250,
-            width: 250
+            height: 280,
+            width: 300
         });
 
         var config = Ext.apply({
             layout:'form',
             cls:'search-filter-panel filter-selection-panel',
+            defaults: {
+                style: {
+                    margin: '2px'
```

Comment:
This is the better way to go in this particular case I think. I prefer to have less CSS rules created as they sometimes have adverse outcomes. Very specific CSS rules also go stale as you saw not too long ago. Also using Id's are not the best way to go as we latter shuffle code